core/test/fixtures/fraggle-rock/artifacts/step0/artifacts.json

```diff
@@ -6,140 +6,160 @@
       "entryType": "measure",
       "startTime": 0,
       "duration": 1,
-      "detail": null
+      "detail": null,
+      "gather": true
```

Comment:
@adamraine are these expected changes? From running update on just `Inputs`.

Reply:
Hmm looks like a bug in `update-flow-fixtures.js`. `Timing` should only update if you specify it.

I'll look into it but we can probably still include it in this PR since that's the artifact we would be storing.

Reply:
Never mind, I think this is expected. `loadArtifacts` adds it when we pull the artifacts from disk:

https://github.com/GoogleChrome/lighthouse/blob/e154f3dcf6c5f2ef0e352846aa50788fa5d3b62e/core/lib/asset-saver.js#L88-L90

I think it makes sense to keep.